docker-images/kafka-based/kafka/scripts/kafka_readiness_kraft.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+set -e
+
+file=/tmp/strimzi.properties
+test -f $file
```

Comment:
do we really need to declare a variable to test the file above?

Reply:
thank for pointing it out! I meant to use it on line 6, so will fix it.